packages/zosjobs/__tests__/__unit__/MonitorJobs.test.ts

```diff
@@ -1,1193 +0,0 @@
-                    {jobname: "FAKE", jobid: "FAKE", status: "OUTPUT"}).then((response) => {
-                    done("Monitor jobs should not have fulfilled the promise because getJobs should have throw and error");
-                }).catch((error) => {
-                    expect(error).toBeDefined();
-                    expect(error instanceof ImperativeError).toBe(true);
-                    expect(error.message).toMatchSnapshot();
-                    expect(mockedGetJobsCommon).toHaveBeenCalledTimes(1);
-                    done();
-                });
-            });
-        });
-
-        describe("polling", () => {
-            it("should expire after the specified number of max attempts", async () => {
-                const attempts: number = 10;
-
-                // Mock GetJobs.getStatusCommon
-                const mockedGetJobsCommon = jest.fn(async (args): Promise<any> => {
-                    return {jobname: "FAKE", jobid: "FAKE", status: "INPUT"};
-                });
-                GetJobs.getStatusCommon = mockedGetJobsCommon;
-
-                let error;
-                try {
-                    const response = await MonitorJobs.waitForStatusCommon(new Session({hostname: "fake", port: 443}),
-                        {jobname: "FAKE", jobid: "FAKE", status: "ACTIVE", attempts, watchDelay: 1});
-                } catch (e) {
-                    error = e;
-                }
-                expect(error).toBeDefined();
-                expect(error instanceof ImperativeError).toBe(true);
-                expect(error.message).toMatchSnapshot();
-                expect(mockedGetJobsCommon).toHaveBeenCalledTimes(attempts);
-            });
-
-            it("should expire after the specified number of max attempts and use the default watch delay", async () => {
-                const attempts: number = 2;
-
-                // Mock GetJobs.getStatusCommon
-                const mockedGetJobsCommon = jest.fn(async (args): Promise<any> => {
-                    return {jobname: "FAKE", jobid: "FAKE", status: "INPUT"};
-                });
-                GetJobs.getStatusCommon = mockedGetJobsCommon;
-
-                let error;
-                try {
-                    const response = await MonitorJobs.waitForStatusCommon(new Session({hostname: "fake", port: 443}),
-                        {jobname: "FAKE", jobid: "FAKE", status: "ACTIVE", attempts});
-                } catch (e) {
-                    error = e;
-                }
-                expect(error).toBeDefined();
-                expect(error instanceof ImperativeError).toBe(true);
-                expect(error.message).toMatchSnapshot();
-                expect(mockedGetJobsCommon).toHaveBeenCalledTimes(attempts);
-            });
-
-            it("should expire after the max attempts and the total time should exceed the attempts multiplied by the delay", async () => {
```

Comment:
Has this test been migrated? I don't see an equivalent one in `MonitorJobs.unit.test.ts`